src/Types.jl

```diff
@@ -259,6 +260,31 @@ function find_project_file(env::Union{Nothing,String}=nothing)
      return project_file
 end
 
+mutable struct Project
```

Comment:
What about having an `other::Dict{String,Any}` field to carry around anything else that might be in the file?

Reply:
Done